app/src/main/java/com/alphawallet/app/entity/tokens/Token.java

```diff
@@ -108,10 +118,11 @@ protected Token(Parcel in) {
         lastTxCheck = in.readLong();
         lastTxUpdate = in.readLong();
         lastTxTime = in.readLong();
-        int readTS = in.readByte();
-        int readDTS = in.readByte();
-        hasTokenScript = readTS == 1;
-        hasDebugTokenscript = readDTS == 1;
+        hasTokenScript = in.readByte() == 1;
+        hasDebugTokenscript = in.readByte() == 1;
+        nonIconifiedWebviewHeight = in.readInt();
```

Comment:
Is this handled for a given token?

1. Has existing TokenScript 
2. Computed height
3. Change TokenScript so that the iconified view has a different height
4. View is rendered with the new height

Reply:
This is so when we pass the token between processes anything setup for that token is passed to that process.

However I think you're asking - do I cache the height - I thought about that - needs to be attached to the hash of the script.

Reply:
> However I think you're asking - do I cache the height - I thought about that - needs to be attached to the hash of the script.

@JamesSmartCell yes :)

iOS caches the height based on the script hash and will refresh with the updated height, but unfortunately, it wouldn't work correctly if the view changes height depending on its contents. But I'll just leave it (iOS) for now, because performance is quite bad on it with heavier views without caching.